ui/page/dexclient/confirm_register_modal.go

```diff
@@ -0,0 +1,136 @@
+package dexclient
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"decred.org/dcrdex/client/asset"
+	"decred.org/dcrdex/client/core"
+	"gioui.org/layout"
+	"gioui.org/widget"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const confirmRegisterModalID = "confirm_register_modal"
+
+type confirmRegisterModal struct {
+	*load.Load
+	modal            *decredmaterial.Modal
+	register         decredmaterial.Button
+	appPassword      decredmaterial.Editor
+	isSending        bool
+	dex              *core.Exchange
+	cert             []byte
+	selectedFeeAsset string
+	confirmed        func()
+}
+
+func newConfirmRegisterModal(l *load.Load, dex *core.Exchange, cert []byte, selectedFeeAsset string) *confirmRegisterModal {
+	md := &confirmRegisterModal{
+		Load:             l,
+		modal:            l.Theme.ModalFloatTitle(),
+		appPassword:      l.Theme.EditorPassword(new(widget.Editor), "App password"),
+		register:         l.Theme.Button("Register"),
+		dex:              dex,
+		cert:             cert,
+		selectedFeeAsset: selectedFeeAsset,
+	}
+
+	md.register.TextSize = values.TextSize12
+	md.register.Background = l.Theme.Color.Primary
+	md.appPassword.Editor.SingleLine = true
+
+	return md
+}
+
+func (md *confirmRegisterModal) ModalID() string {
+	return confirmRegisterModalID
+}
+
+func (md *confirmRegisterModal) Show() {
+	md.ShowModal(md)
+}
+
+func (md *confirmRegisterModal) Dismiss() {
+	md.DismissModal(md)
+}
+
+func (md *confirmRegisterModal) OnDismiss() {
+	md.appPassword.Editor.SetText("")
+}
+
+func (md *confirmRegisterModal) OnResume() {
+	md.appPassword.Editor.Focus()
+}
```

Comment:
I believe we can use the `NewPasswordModal()` for this. 
A functions can be added to collect the ` dex *core.Exchange` and  `cert []byte` parameters.

Kindly have this modal removed.

Reply:
This is already sorted in the `NewPasswordModal()` modal so we don't need this one.